src/test/java/io/arlas/server/rest/tag/UnTagIT.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.rest.tag;
+
+import io.arlas.server.AbstractTestWithCollection;
+import io.arlas.server.CollectionTool;
+import io.arlas.server.model.request.Tag;
+import io.arlas.server.model.request.TagRequest;
+import io.arlas.server.rest.explore.SearchServiceIT;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.StringDescription;
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.function.Function;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+public class UnTagIT extends AbstractTestWithCollection {
```

Comment:
To stay consistent with other IT classes, UnTagIT and TagIT should be merged in a TagServiceIT using `getUrlPath(CollectionTool.COLLECTION_NAME)+"/_tag"` or `getUrlPath(CollectionTool.COLLECTION_NAME)+"/_untag"` urls. On top of that, it would avoid to instantiate `new TagIT()` useless objects.

Reply:
ok